MVVM-C/AppDelegate.swift

```diff
@@ -10,15 +10,11 @@ import UIKit
 
 @UIApplicationMain
 
-class AppDelegate: UIResponder, UIApplicationDelegate
-{
+class AppDelegate: UIResponder, UIApplicationDelegate {
```

Comment:
Let's add `final` (it's generally a good idea when you don't expect inheritance).